### PR TITLE
Add support for refresh interval

### PR DESCRIPTION
**Pull request by hurnhu**
Added code to change the refresh interval.
Some UI change might need to be made, but functionality is there.
#1

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -104,6 +104,18 @@
...
+				<div class="col-8">
+					<div class="input-group mb-3">
+						<input type="number" id="refreshRate" class="form-control" value="10">
+						<div class="input-group-append">
+							<span class="input-group-text">Seconds</span>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
```
Comment (TwinProduction) [inline static/index.html:118]: Perhaps I'm a bit picky, but I feel like a small drop-down aligned to right right would be cuter.

**Review by TwinProduction — CHANGES_REQUESTED** [review]

File: static/index.html (hunk 1 above)
Comment (hurnhu) [reply static/index.html]: Suggestions have been added

**Comment by hurnhu** [comment]
I've changed it to a select drop-down and aligned it right.
On load the select is populated (up to 600) for available options.

It might be a good idea to add another issue for another drop down to select seconds or minutes for the refresh rate.

**Review by hurnhu — COMMENTED** [review]
awesome

**Comment by TwinProduction** [comment]
I've made some minor adjustments, I hope you don't mind.

I felt that having the refresh rate at in the main container made it feel a bit out of place, and since the main purpose of the dashboard are the status of the services themselves, positioning the refresh interval setting in a more subtle location would give an overall better experience.

What do you think? @hurnhu

**Review by TwinProduction — APPROVED** [review]

**Comment by TwinProduction** [comment]
Released in v1.1.1

**Pull request merged**